godot-macros/src/docs.rs

```diff
@@ -225,8 +277,8 @@ fn format_params_xml<'a, 'b>(
 }
 
 fn format_virtual_method_xml(method: venial::Function) -> Option<String> {
-    let desc = attribute_docs_to_bbcode(&method.attributes)?;
-    let desc = xml_escape(desc);
+    // Note: Virtual methods can't be deprecated.
+    let (desc, _, _) = attribute_docs_to_bbcode(&method.attributes)?;
```

Comment:
Good note, but use of unsupported annotations should probably cause an error for the user. 

From UX perspective, it's frustrating to type a `@experimental` or `@deprecated` paragraph, it not showing up in the editor, and having to investigate if it's their fault, godot-rust's fault or Godot's fault.